client/src/components/SupportTools/SupportTools.tsx

```diff
@@ -109,6 +114,20 @@ const Column = styled.div`
   padding-right: 30px;
 `
 
+const Column1 = styled.div`
+  flex: 0 0 25%;
+  padding-right: 30px;
```

Comment:
Instead of using padding here, I'd add a `gap` to the parent `Row`. A little easier to manage. Also with that change you could probably just get away with inline styles for the flex basis instead of defining styled components for each column

Reply:
Finding that using `gap` is getting a bit off because it isn't accounted for when the flex basis is calculated, so the total width goes over 100% when using them together. But agreed on making the styles inline instead of styled components